AdminWebsite/AdminWebsite.AcceptanceTests/Hooks/DriverHooks.cs

```diff
@@ -67,12 +67,11 @@ public void SetTimeZone(TestContext context)
             context.TimeZone = new TimeZone(context.WebConfig.SauceLabsConfiguration.RunningOnSauceLabs(), context.WebConfig.TestConfig.TargetOS);
         }
 
-        private static bool EnableLogging(ScenarioInfo scenario)
+        private static bool EnableLogging()
         {
-            return !scenario.Tags.Contains("DisableLogging");
+            return false;
```

Comment:
Also I am disabling logging as part of the AC tests to see if it fixes a strange Sauce Labs issue. @robin-weller is aware